app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -184,7 +184,9 @@ public void onClick(View view) {
                         Globals.EventLogger.LogData(Constants.LOGKEY_SCOUTER, preMatchBinding.editScouterName.getText().toString().toUpperCase());
                         Globals.EventLogger.LogData(Constants.LOGKEY_DID_PLAY, String.valueOf(preMatchBinding.checkboxDidPlay.isChecked()));
                         Globals.EventLogger.LogData(Constants.LOGKEY_TEAM_SCOUTING, String.valueOf(Globals.CurrentScoutingTeam));
-                        Globals.EventLogger.LogData(Constants.LOGKEY_START_POSITION, String.valueOf(Globals.StartPositionList.getStartPositionId(preMatchBinding.spinnerStartingPosition.getSelectedItem().toString())));
+                        if (checkbox_DidPlay.isChecked())
+                            Globals.EventLogger.LogData(Constants.LOGKEY_START_POSITION, String.valueOf(Globals.StartPositionList.getStartPositionId(spinner_StartPos.getSelectedItem().toString())));
+                        else Globals.EventLogger.LogData(Constants.LOGKEY_START_POSITION, "Did Not Play");
```

Comment:
Let's not hard code "Did Not Play".  we'll know in Tableau they didn't play.  You really don't need to log anything.  The Logger will know there's no value and output an empty string.